haiku-apps/sync_modular/sync_modular-2.2.1b.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="Native, modular software synthesizer"
+DESCRIPTION="It gives you, though limited as any synth, but quite versatile possibilities for creating \
```

Comment:
That's not a nice first sentence. How about going with what's on its website:
"Sync modular is a modular synthesizer allowing you to build your own synthesizer and sounds. In 2010, Sync modular became a freeware. You can use this name and serial number to register the application:

User name : Free registration
Serial number : eCDiPa-kbbmPZ-zGDJuyU

You can download instruments and macros created by users from SyncModular's website.